scripts/ezfio_interface/qp_convert_output_to_ezfio.py

```diff
@@ -309,8 +309,16 @@ def parse_str(pseudo_str):
             array_l_max_block.append(l_max_block)
             array_z_remove.append(z_remove)
 
-            matrix.append([[coef_n_zeta.split()[1:] for coef_n_zeta in l.split('\n')] for l in array_party[1:]])
-
+            x = [[coef_n_zeta.split() for coef_n_zeta in l.split('\n')] \
+               for l in array_party[1:] ]
+            x = []
```

Comment:
Changed:
            x = [[ filter(None,coef_n_zeta.split()) for coef_n_zeta in l.split('\n')] for l in array_party[1:] ]